<launch>

	<group ns="image_pub">
		<node pkg="image_processing_hw" name="image_pub" type="image_pub_all.py"/>
	</group>
	<group ns="hw7">
		<node pkg="hw2_package" name="hw7_node" type="imaging.py"/>
	</group>

	<group ns="hw8">
		<node pkg="hw8_lane_detection" name="lane_detecting" type="lane_detect.py"/>

		<node pkg="rqt_graph" name="rqt_graph" type="rqt_graph" />
		<node pkg="rqt_console" name="rqt_console" type="rqt_console" />

		<node pkg="rqt_image_view" name="image_view_cropped" type="rqt_image_view" args="/image_edges"/>
		<node pkg="rqt_image_view" name="image_view_white" type="rqt_image_view" args="/image_lines_white"/>
		<node pkg="rqt_image_view" name="image_view_yellow" type="rqt_image_view" args="/image_lines_yellow"/>
	</group>
	<node pkg="rostopic" name="rostopic" type="rostopic" output="screen" args="echo /image"/>
</launch>